docs/launching-apps/prerequisites.rst

```diff
@@ -27,6 +34,214 @@ example:
                ``ssh``-based logins will be able to find the Open MPI
                executables and libraries.
 
+               For example, if Open MPI was installed with a prefix of
+               ``/opt/openmpi``, then the following should be in your
+               ``PATH`` and ``LD_LIBRARY_PATH``
+
+               .. list-table::
+                  :header-rows: 1
+
+                  * - Environment variable
+                    - Value to add
+
+                  * - ``PATH``
+                    - ``/opt/openmpi/bin``
+
+                  * - ``LD_LIBRARY_PATH``
+                    - ``/opt/openmpi/lib``
+
+               Depending on your environment, you may need to set these
+               values in your shell startup files (e.g., ``.bashrc``,
+               ``.cshrc``, etc.).
+
+Additionally, Open MPI requires that jobs can be started on remote
+nodes without any input from the keyboard.  For example, if using
+``ssh`` as the remote agent, you must have your environment setup to
+allow execution on remote nodes without entering a password or
+passphrase.
+
+Adding Open MPI to ``PATH`` and ``LD_LIBRARY_PATH``
```

Comment:
This is all correct, but also a bit confusing because we now runpath on most of our supported platforms, meaning that the LD_LIBRARY_PATH bits aren't necessarily required.

Reply:
Updated to clarify that runpath should be taking care of locating the Open MPI executables and libraries, but if that's not working, then drop into the discussion about setting the PATH and LD_LIBRARY_PATH environment variables.  Included in next push.